website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,241 @@
+first time, you will also need to [compelete the bootstrapping process](/consul/security-networking/production-acls).
+
+## Create the Replication Token for ACL Management
+
+Next, create the replication token for managing ACLs. The token should be
+created with the following privileges.
+
+- acl = "write" which will allow you to replicate tokens.  
+- operator = "read" for replicating proxy-default configuration entries.
+- service_prefix, policy = "read" and intentions = "read" for replicating
+service-default configuration entries, CA, and intention data. 
+
+```hcl 
+acl = "write"
+
+operator = "read"
+
+service_prefix "" { 
+  policy = "read" 
+  intentions = "read" 
+} 
+```
+
+Now that you have the ACL rules defined, create a policy with those rules. 
+
+```sh 
+$ consul acl policy create -name replication -rules @replication-policy.hcl 
+ID:           240f1d01-6517-78d3-ec32-1d237f92ab58
+Name:         replication 
+Description: Datacenters: 
+Rules: acl = "write"
+
+operator = "read"
+
+service_prefix "" { policy = "read" intentions = "read" } 
+```
+
+Finally, use your newly created policy to create the replication token.
+
+```sh 
+$ consul acl token create description "replication token" -policy-name replication 
+AccessorID:   67d55dc1-b667-1835-42ab-64658d64a2ff 
+SecretID:     fc48e84d-3f4d-3646-4b6a-2bff7c4aaffb 
+Description:  replication token 
+Local:        false
+Create Time:  2019-05-09 18:34:23.288392523 +0000 UTC 
+Policies: 
+  240f1d01-6517-78d3-ec32-1d237f92ab58 - replication 
+```
+
+## Enable ACL Replication on the Secondary Datacenter
+
+Once you have configured the primary datacenter and created the replication
+token, you can setup the secondary datacenter. 
+
+-> Note, your initial `bootstrap` token can be used for the necessary
+privileges to complete any action on the secondary servers. 
+
+### Configure the Servers 
+
+You will need to set the `primary_datacenter` parameter to the name of your
+primary datacenter and `enable_token_replication` to true.  
+
+```json 
+{ 
+"datacenter": "dc_secondary", 
+"primary_datacenter": "primary_dc", 
+"acl": { 
+  "enabled": true, 
+  "default_policy": "deny", 
+  "down_policy": "extend-cache", 
+  "enable_token_persistence": true,
+  "enable_token_replication": true
+  } 
+} 
+```
+
+Now you can start the agent.
+
+```sh 
+$ consul agent -config-file=server.json 
+``` 
+
+### Apply the Replication Token to the Servers
+
+Finally, apply the replication token to all the servers using the CLI. 
+
+```sh 
+$ consul acl set-agent-token replication <token> 
+ACL token "replication" set successfully 
+```
+
+Once token replication has been enabled, you will also be able to create
+datacenter local tokens.
+
+Repeat this process on all servers. If you are configuring ACLs for the
+first time, you will also need to [set the agent token](/consul/security-networking/production-acls#add-the-token-to-the-agent).
+
+Note, the clients do not need the replication token.
```

Comment:
Should this be a Note-style call out?

Reply:
I try to reserve the call outs for really important information. If you applied or tried to apply the replication token to the client it shouldn't be detrimental to the cluster.